tcf_core/auth_pipeline.py

```diff
@@ -44,24 +45,50 @@ def password_validation(backend, details, request, response, *args, **kwargs):
             return redirect("/login/password_error", error=True)
     return {"password": response.get("password")}
 
+# Make BaseStrategy.validate_email() always return True, so that
+# social_core.pipeline.mail.mail_validation can be run multiple times
+BaseStrategy.validate_email = lambda *args, **kwargs: True
```

Comment:
This is needed because the `validate_email` method prevents `social_core.pipeline.mail.mail_validation` from being able to run more than once.